### PR TITLE
Release 4.0.5

### PR DESCRIPTION
**Pull request by melting-snowman**
Release latest minor instalment of the SDK.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 
 #### 4.x Releases
 
+## [4.0.5](https://github.com/checkout/frames-ios/releases/tag/4.0.5)
+
+Released on 2023-01-05
+
+Updates:
+
+- Add updates
```
Comment (alex-ioja-yang-cko) [inline CHANGELOG.md:14]: We need to modify this before merging.

Comment (patrick-hoban-cko) [inline CHANGELOG.md:14]: Updates:

- Fixed a minor bug with the pay button not becoming disabled once an expiry date is invalid
- Improved UI test cases for the expiry date field to be more extensible
- Updated phone number validation logic, to append the country code to the user's selected country

**Review by okhan-okbay-cko — APPROVED** [review]
🥎

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
...
 #### 4.x Releases
 
+## [4.0.5](https://github.com/checkout/frames-ios/releases/tag/4.0.5)
+
+Released on 2023-05-24
+
+Updates:
+
+- Fixed a minor bug with the pay button not enabling due to inconsistent behaviour with expiry date formatting
+- Improved UI test support, enabling public accessibilityLabels to input fields
```
[the remark this answers is not in the document]
Comment (alex-ioja-yang-cko) [reply CHANGELOG.md:15]: I will keep the current format as whilst I did mistakenly pluralise it, it does hint correctly at how to find it 😄

**Pull request merged**